csl.rnc

```diff
@@ -152,171 +150,173 @@ div {
     ## should be stable, unique and dereferenceable URI.
     element cs:id { xsd:anyURI }
-  
+
   info.issn =
-    
+
     ## Specify the journal's ISSN(s) for journal-specific styles. An ISSN
     ## must consist of four digits, a hyphen, three digits, and a check
     ## digit (a numeral digit or roman X), e.g. "1234-1231".
     element cs:issn { issn }
-  
+
   info.eissn =
-    
+
     ## Specify the journal's eISSN for journal-specific styles.
     element cs:eissn { issn }
-  
+
   info.issnl =
-    
+
     ## Specify the journal's ISSN-L for journal-specific styles.
     element cs:issnl { issn }
-  
+
   issn = xsd:string { pattern = "\d{4}\-\d{3}(\d|x|X)" }
-  
+
   independent-style.info.link =
     element cs:link {
       attribute href { xsd:anyURI },
-      
+
       ## Specify how the URL relates to the style.
       attribute rel {
-        
+
         ## The URI of the CSL style itself.
         "self"
-        | 
+        |
           ## URI of the style from which the current style is derived.
           "template"
-        | 
+        |
           ## URI of style documentation.
           "documentation"
-        | 
+        |
           ## Obsolete for independent styles. Will be disallowed with
           ## CSL 1.1.
           "independent-parent"
       },
       info-text
     }
-  
+
   dependent-style.info.link =
     element cs:link {
       attribute href { xsd:anyURI },
-      
+
       ## Specify how the URL relates to the style.
       attribute rel {
-        
+
         ## The URI of the CSL style itself.
         "self"
-        | 
+        |
           ## URI of the CSL style whose content should be used for
           ## processing. Required for dependent styles.
           "independent-parent"
-        | 
+        |
           ## URI of style documentation.
           "documentation"
-        | 
+        |
           ## Obsolete for dependent styles. Will be disallowed with CSL
           ## 1.1.
           "template"
       },
       info-text
     }
-  
+
   info.published =
-    
+
     ## Specify when the style was initially created or made available.
     element cs:published { xsd:dateTime }
-  
+
   info.rights =
     element cs:rights {
       attribute license { xsd:anyURI }?,
       info-text
     }
-  
+
   info.summary = element cs:summary { info-text }
-  
+
   info.title = element cs:title { info-text }
-  
+
   info.title-short =
-    
+
     ## Specify an abbreviated style title (e.g., "APA")
     element cs:title-short { info-text }
-  
+
   info.updated =
-    
+
     ## Specify when the style was last updated (e.g.,
     ## "2007-10-26T21:32:52+02:00")
     element cs:updated { xsd:dateTime }
-  
+
   info-text =
     attribute xml:lang { xsd:language }?,
     text
 }
-[ a:documentation [ xhtml:h2 [ "cs:locale in Independent Styles" ] ] ]
+
+#==============================================================================
+
+## cs:locale in Independent Styles
```

Comment:
This could be a separate `csl-locale.rnc` file.

Reply:
Note that we have `locale-file.locale` (root element of CSL locale files) and `style.locale` (locale section of CSL styles) definitions, so this doesn't seem like the right place for a split.